### PR TITLE
CommentToPhpdocFixer - allow to ignore tags

### PR DESCRIPTION
**Pull request by kubawerlos**
Resolves https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3628

Ping @julienfalque for review, I've made the list case insensitive - I figured it's more natural, would you agree?

### CODE REVIEW

**Comment by keradus** [comment]
ping @julienfalque , can you take a look please?

File: tests/Fixer/Comment/CommentToPhpdocFixerTest.php [hunk 1]
```diff
@@ -246,6 +248,70 @@ public function provideTestCases()
 EOT
                 ,
             ],
+            [
+                <<<'EOT'
+<?php /* header comment */ $foo = true;
+
+// @todo do something later
+$foo = 1;
```
Comment (julienfalque) [inline tests/Fixer/Comment/CommentToPhpdocFixerTest.php]: Please add a test case with a comment before a structural element, I think the added tests here would still pass without the `ignored_tags`, thus they don't prove the option works as expected.

Comment (kubawerlos) [reply tests/Fixer/Comment/CommentToPhpdocFixerTest.php]: What do you mean? Isn't this case with a comment before a structural element? Can you prepare the one you are thinking about?

Comment (julienfalque) [reply tests/Fixer/Comment/CommentToPhpdocFixerTest.php]: Actually I misunderstood the code, sorry.

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**